Dockerfile

```diff
@@ -4,7 +4,17 @@ USER root
 RUN apt-get update -y \
     && apt-get install -y openjdk-8-jdk \
     && apt-get install -y curl \
-    && apt-get install -y git
+    && apt-get install -y git \
+    && apt-get install -y sudo \
+    && apt-get install -y python-pip \
+    && apt-get install -y openssh-server
+
+RUN mkdir /var/run/sshd
+ADD ssh/jc_rsa.pub /root/.ssh/jc_rsa.pub
+RUN cat root/.ssh/jc_rsa.pub >> ~/.ssh/authorized_keys
+RUN chmod go-w ~/ && chmod 700 ~/.ssh && chmod 600 ~/.ssh/authorized_keys
```

Comment:
nit: can you combine the RUN statements? Each individual one increases overall container size